demo/run.sh

```diff
@@ -0,0 +1,6 @@
+export UAVCAN__SUB__TEMPERATURE_SETPOINT__ID=2345
```

Comment:
This file should not be here, you can just run the orc file to the same effect.

Reply:
For some reason `yakut orc` crashes msys bash terminal on windows, but I didn't have time to investigate. I will remove the file

Reply:
Sorry, the orchestrator is incompatible with Windows indeed :D I thought you are on GNU/Linux.

Reply:
It uses bash commands so it should work in msys terminal (git bash), but something goes really wrong and entire terminal crashes 🤷